custom_components/places/coordinator.py

```
@@ -0,0 +1,511 @@
+            attributes.update(
+                {
+                    attr: self.get_attr(attr)
+                    for attr in EXTENDED_ATTRIBUTE_LIST
+                    if not self.is_attr_blank(attr)
+                }
+            )
+        return attributes
+
+    def _initialize_config_attributes(self) -> None:
+        """Seed runtime attributes from static config entry data."""
+        self.set_attr(ATTR_INITIAL_UPDATE, True)
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)